packages/graphql-typescript-definitions/package.json

```diff
@@ -25,7 +25,6 @@
     "prepublishOnly": "yarn build"
   },
   "devDependencies": {
-    "@types/chalk": "^2.2.0",
```

Comment:
We don't need `@types/chalk` finally? 😃

Reply:
It's a stub now. Chalk includes its own types.